inim.nimble

```diff
@@ -13,6 +13,3 @@ requires "nim >= 1.0.0"
 requires "cligen >= 0.9.15"
 requires "noise"
 
-
-task test, "Run all tests":
-  exec "nim c -d:prompt_no_history tests/test.nim"
```

Comment:
The `prompt_no_history` hack to get `noise` passing tests is no longer needed, as it's now in a nim.cfg in the `tests/` directory, as it should be.